tests/framework/components/test_component.py

```diff
@@ -374,44 +386,43 @@ def test_component_lookup_table_configuration(hdf_file_path):
     for key, data in artifact_data.items():
         artifact.write(key, data)
 
-    component = ColumnCreator()
+    component = LookupCreator()
     sim = InteractiveContext(components=[component], setup=False)
     sim.configuration.update(
         {
             "input_data": {"artifact_path": hdf_file_path},
             component.name: {
-                "lookup_tables": {
-                    "favorite_team": Component.build_lookup_table_config(
-                        value="data",
-                        categorical_columns=["test_column_1"],
-                        continuous_columns=[],
-                        key_name="simulants.favorite_team",
-                    ),
-                    "favorite_scalar": Component.build_lookup_table_config(
-                        value=0.4,
-                    ),
-                    "favorite_color": Component.build_lookup_table_config(
-                        value="data",
-                        categorical_columns=["test_column_2"],
-                        continuous_columns=["test_column_3"],
-                        key_name="simulants.favorite_color",
-                    ),
-                    "baking_time": Component.build_lookup_table_config(
-                        value="data",
-                        categorical_columns=["test_column_1", "test_column_2"],
-                        continuous_columns=["test_column_3"],
-                        key_name="simulants.baking_time",
-                        skip_build=True,
-                    ),
-                    "favorite_number": Component.build_lookup_table_config(
-                        value="data",
-                        categorical_columns=[],
-                        continuous_columns=["test_column_3"],
-                        key_name="simulants.favorite_number",
-                    ),
-                },
+                "favorite_team": Component.build_lookup_table_config(
```

Comment:
Since you did end up creating a class for this test (`LookupCreator()`), I think you should define the configs in the class itself.

Reply:
Ok yea I had to overwrite the property so I will also do that.